src/lib/server/services/content.ts

```diff
@@ -268,6 +270,27 @@ export const getPostStatistics = async (db: DB) => {
 };
 export type PostStatistics = Awaited<ReturnType<typeof getPostStatistics>>;
 
+export const createCommunityReport = async (
+    db: DB,
+    newPostCommunityReport: NewPostReport
+): Promise<PostReport> => {
+    const [ret] = await db
+        .insert(channelPostReportTable)
+        .values(newPostCommunityReport)
+        .returning();
+
+    return ret;
+};
+
+export const createSiteReport = async (
+    db: DB,
+    newPostSiteReport: NewPostReport
```

Comment:
Can you change the name of this argument to be `newPostReport` to keep it consistent across the site?